es-app/src/SystemData.cpp

```diff
@@ -60,7 +60,7 @@ void SystemData::setIsGameSystemStatus()
 	// we exclude non-game systems from specific operations (i.e. the "RetroPie" system, at least)
 	// if/when there are more in the future, maybe this can be a more complex method, with a proper list
 	// but for now a simple string comparison is more performant
-	mIsGameSystem = (mName != "retropie");
+	mIsGameSystem = (mName != "retropie") && (mName != "kodi");
```

Comment:
Ah, certainly. That's a valid use case.

The way it currently works is:
1 - If on the System View screen, clicking the "Random" button will show a different random system (staying on the system view);
2 - If on the Gamelist View, it will select a different entry each time.

The way this is used for the random logic in particular is so that, in case 1, when in the System View, it won't take you to one of the non-game systems entries as the random choice. 

If someone would want to select a random video from within the Kodi system, opening the Kodi system and pressing "random" will certainly show him different entries there. It just won't happen that, for someone looking for a random suggestion of a system to play, they'll get a suggestion for "RetroPie settings" or "Kodi" - that was the intention.

I've removed it for now, as it's a small enough thing that the user just needs to press "random" again, but let me know if after the clarification you'd rather have me put it back. 

Or I can re-add it in a future Collections-related PR, in order not to delay this PR anyway.

Thanks for reading through the code!